ci-operator/step-registry/openshift/e2e/test/qe/long-duration/openshift-e2e-test-qe-long-duration-chain.yaml

```diff
@@ -1,6 +1,7 @@
 chain:
   as: openshift-e2e-test-qe-long-duration
   steps:
+  - ref: openshift-extended-test-longduration
```

Comment:
For this special use case, perhaps we should run `openshift-extended-test-longduration` after `cucushift-e2e-long-duration`, given cucushift runs for 30m40s, and ginkgo runs for 7h57m30s, so that we do not need to wait for 8 hours before we can see cucushift results.

Reply:
@liangxia Thanks, I understand your concern. 
The golang testing is becoming focus, so I would like to run golang case on refresh cluster firstly to get clean result. I guess it should be key concern from release manager
About if we could get cucushift case result quickly before test is done, it should not be key concern from release manager's view.

So, I would like to keep golang before cucushifit here.

@chengzhang1016 @jhou1 could you please share your opinion here? thanks